blocks/editable/index.js

```diff
@@ -72,6 +73,10 @@ function getFormatProperties( formatName, parents ) {
 
 const DEFAULT_FORMATS = [ 'bold', 'italic', 'strikethrough', 'link' ];
 
+const AUTO_INLINE_MODE_TAG_GROUPS = [
```

Comment:
Since this is paste specific, I'm wondering if maybe we can't merge this with the whitelist into some sort of config for the raw handling. We'd just supply `rawHandler` with a base tag. Thoughts?

Reply:
So we would pass the tag to rawHandler and rawHandler would build the array using AUTO_INLINE_MODE_TAG_GROUPS right?
The only disadvantage I see in this approach is that we lose the possibility to pass a custom array in editable, maybe it will be useful for some block. But right now we don't have a use case for that so I'm fine with that change :)

Reply:
I was also thinking this would live along with the whitelist stuff and `rawHandler` would only care about the one tag, yes.

Reply:
Hi @mcsf and @iseulde some changes were applied that move things in that direction. Feel free to a have a new look and let me your thoughts. Thank you for your reviews.